src/System.Linq/tests/ConcatTests.cs

```diff
@@ -151,6 +152,52 @@ public static IEnumerable<object[]> ConcatWithSelfData()
 
         public static IEnumerable<object[]> ChainedCollectionConcatData() => GenerateSourcesData(innerTransform: e => e.ToList());
 
+        public static IEnumerable<object[]> AppendedPrependedConcatAlternationsData()
+        {
+            var @base = Array.Empty<int>();
```

Comment:
Nit: here and in future PRs, please avoid using keywords.  There are plenty of other names to choose from that don't require the @ sign :wink: